lib/msal-core/karma.conf.js

```diff
@@ -7,16 +7,23 @@ module.exports = function (config) {
         frameworks: ['mocha', 'chai'],
         files: [
             'node_modules/babel-polyfill/dist/polyfill.js',
-            'tests/mochaTests/**/*.ts',
+            'tests/**/*.ts',
+        ],
+        exclude: [
+            'tests/MSAL*',
```

Comment:
should this have a file extension?

Reply:
this is just temporary, this will be gone in the future.